### PR TITLE
sql/logic_test: avoid wildcards

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: sql/create.go [hunk 1]
```diff
@@ -121,11 +120,14 @@ func (p *planner) CreateIndex(n *parser.CreateIndex) (planNode, error) {
 	var b client.Batch
 	b.Put(MakeDescMetadataKey(tableDesc.GetID()), tableDesc)
 
+	// dummy write so indexing works later
```
Comment (petermattis) [inline sql/create.go]: Can you expand on this comment? Why would you need to index into `writes` if there were no writes.

**Comment by petermattis** [comment]
LGTM

File: sql/create.go (hunk 1 above)
Comment (tamird) [inline sql/create.go]: done, ptal

File: sql/create.go [hunk 2]
```diff
@@ -121,11 +120,17 @@ func (p *planner) CreateIndex(n *parser.CreateIndex) (planNode, error) {
 	var b client.Batch
 	b.Put(MakeDescMetadataKey(tableDesc.GetID()), tableDesc)
 
+	// We need a dummy element in this slice to account for the `Put`
+	// above; the `Put` won't fail, but if we don't add an entry for it,
+	// we'll end up with out-of-bounds error when we iterate over the
+	// writes in the error case.
```
Comment (petermattis) [inline sql/create.go]: Nice.

**Pull request merged**